www/js/components/add-point-location.js

```diff
@@ -12,6 +12,8 @@ export class AddPointLocation extends Component {
 
   constructor(props) {
     super(props);
+    const { dispatch , mapState } = this.props;
+    dispatch(setPointLocation(mapState.center));
```

Comment:
we do this dispatch on componentDidMount, it should not be here in the constructor